infoschema/infoschema.go

```diff
@@ -358,5 +358,13 @@ func initInfoSchemaDB() {
 
 // IsMemoryDB checks if the db is in memory.
 func IsMemoryDB(dbName string) bool {
-	return dbName == "information_schema" || dbName == "performance_schema"
+	if dbName == "information_schema" {
+		return true
+	}
+	for _, driver := range drivers {
```

Comment:
when do we call `tiniub.Init()`? and do we have race-condition to access this `drivers`? for current plugin is start time is ok...

but if drivers can be dynamic change it's better to take care race.